src/cosmosdb-preview/azext_cosmosdb_preview/_params.py

```diff
@@ -194,7 +196,7 @@ def load_arguments(self, _):
             'managed-cassandra datacenter create',
             'managed-cassandra datacenter update']:
         with self.argument_context(scope) as c:
-            c.argument('node_count', options_list=['--node-count', '-n'], help="The number of Cassandra virtual machines in this data center.")
+            c.argument('node_count', options_list=['--node-count', '-n'], help="The number of Cassandra virtual machines in this data center. The minimum value is 3.")
```

Comment:
Could you add a validator to validate minimum value?

Reply:
done.